std/digest/murmurhash.d

```diff
@@ -514,7 +514,15 @@ struct MurmurHash3(uint size /* 32 or 128 */ , uint opt = size_t.sizeof == 8 ? 6
         const remainderStart = numElements * Element.sizeof;
         foreach (ref const Element block; cast(const(Element[]))(data[0 .. remainderStart]))
         {
-            putElement(block);
+            version(ARM)
+            {
+                // Since the block may be unaligned, this can trip up ARM
+                // codegen, so place it in a newly aligned Element first.
+                Element alignedTemp = block;
```

Comment:
OK, I'm not familiar with these alignment issues and exactly how D code is likely to trigger them: I just used the simplest fix that worked.  I suppose you want [something more like @kinke's patch](https://github.com/kinke/phobos/commit/83474aeb3ee9bf089ce97c847d9d72816a881ada), which uses different types based on the alignment, though maybe he had problems with certain types that are commented out.